Configurations/release-move-tag.sh

```diff
@@ -4,23 +4,28 @@ set -e
 # Convenience script to automatically commit Package.swift after updating the checksum and move the latest tag
 latest_git_tag=$(git describe --abbrev=0) # gets the latest tag name
```

Comment:
We still have to update this right? Otherwise it's going to get the wrong tag name since we are creating the tag after the workflow? Recall the tag has to be created after this script updates the Package.swift